packages/web-components/src/components/content-section-banner/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,63 @@
+import { Props, Description } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import markdownContents from '../../../../docs/markdown-contents.md';
+
+# `Content section banner`
+
+> The Content Section Banner - a Content Group Banner component assembled in a section with a left-column header.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+##### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/cta/link-list-item-cta.js';
+import '@carbon/ibmdotcom-web-components/es/components/link-list/link-list.js';
+import '@carbon/ibmdotcom-web-components/es/components/content-group/content-group-heading.js';
+import '@carbon/ibmdotcom-web-components/es/components/content-group-banner/content-group-banner.js';
+import '@carbon/ibmdotcom-web-components/es/components/content-section-banner/content-section-banner.js';
+```
+
+### HTML
+
+```html
+<dds-content-section-banner heading="Aliquam condimentum interdum">
```

Comment:
@proeung I'm wondering if this could possibly be either a slot or element instead of being passed as an attribute, since we'd want to have lightDOM be available from an SEO perspective for non Googlebot crawlers.

Reply:
@jeffchew  The `<dds-content-section-heading>` is already defined as a slot. I just pushed up a commit that alters the README file of the `<dds-content-section-banner>` component so that folks understand that they can use this as a slot or as the light DOM with heading tags. See here - https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/5697/commits/e8943a4e71d8b985ea9726989221fad63707600c.